core-bundle/src/Controller/FrontendController.php

```diff
@@ -42,13 +43,13 @@ public function indexAction(): Response
     /**
      * @Route("/_contao/cron", name="contao_frontend_cron")
      */
-    public function cronAction(): Response
+    public function cronAction(Request $request, Cron $cron): Response
     {
-        $this->get('contao.framework')->initialize();
-
-        $controller = new FrontendCron();
+        if (!$request->isMethod('post')) {
```

Comment:
Should we not check `if ($request->isMethod('get'))` instead?

Reply:
🤷‍♂ the old `FrontendCron` controller checked for `!empty($_POST)`. But I agree that checking for `get` instead makes more sense.